setup.py

```diff
@@ -35,7 +35,8 @@ def read(filename):
         'Django>=1.6.2',
         'pytest',
         'mock',
-        'boto>=2.32.0'
+        'boto>=2.32.0',
+        'dropbox>=2.2.0'
```

Comment:
I have correct this by add a ``requirements-test.txt`` and this line in ``setup.py``:

```
...
tests_require=readlines('requirements-tests.txt'),
...
```